sequelize/sequelize
# feat: support typescript >= 4.5's module resolution

## Description of changes

See https://github.com/sequelize/sequelize/pull/14610 for details

This PR is that PR, but with the last change request applied.

Closes #14610

## How to test this

We don't currently have the infrastructure to test how the different versions & configurations of TypeScript import Sequelize

In the meantime, you can clone github.com/sequelize/sequelize-sscce, update `@sequelize/core` to its latest version, then edit the `package.json` configuration of `@sequelize/core` to apply the changes made to `exports`
1
TSC's module resolution option should be set to `nodenext`, `node12`, and/or `node16` (node16 replaces node12 in newer versions of TSC)
